docs/content/concepts/scheduler.md

```diff
@@ -1,55 +1,43 @@
 ---
-title: Load Scheduler
+title: Scheduler
 keywords:
   - scheduler
   - tokens
   - priority
   - queuing
   - actuators
-sidebar_position: 6
+sidebar_position: 7
 ---
 
-:::info
-
-See also
-[_Adaptive Load Scheduler_ reference](/reference/configuration/spec.md#adaptive-load-scheduler)
-and
-[_Load Scheduler_ reference](/reference/configuration/spec.md#adaptive-load-scheduler).
-
-:::
-
-The _Load Scheduler_ is a powerful tool designed to protect your services from
-overloads and prevent cascading failures. Its **primary goal is to limit** the
-number of concurrent requests to a service, ensuring that the service can handle
-the incoming workload. By defining workloads with different priorities and
-weights, the _Load Scheduler_ can prioritize certain requests over others,
-enabling graceful degradation of service during times of high traffic.
+The Scheduler is a crucial component in Aperture, prioritizing requests based on
+their priority and size. It operates based on a
+[Weighted Fair Queue-based](https://en.wikipedia.org/wiki/Weighted_fair_queueing)
+methodology, which ensures the fair handling of workloads, particularly during
+periods of high or overload traffic. By prioritizing critical application
+features over low priority workloads, the scheduler helps optimize user
+experience and maximize good throughput.
+
+Each Agent instantiates an independent copy of the _Scheduler_, but output
+signals for accepted and incoming token rate are aggregated across all agents.
+The Scheduler utilizes a Load Multiplier to determine the [tokens](#tokens)
```

Comment:
LYou ad multiplier is specific to load scheudler

Reply:
>Load multiplier is specific to load scheudler

-Okay!

**Does adding below section makes sense? Based on our previous discussion i came to this conclusion. ** @tanveergill ?


### Load Scheduler {#load-scheduler}
  
  The *Load Scheduler* is a primary actuation component layered atop the base  
  Scheduler. It provides active service protection by forming a queue for flows  
  before they reach the service. Through provided match rules, flows are organized  
  into Workloads, each determining the priority and cost of admitting its  
  associated flows. The *Load Scheduler* controls the incoming tokens per second,  
  which, in concurrency limiting cases, can be interpreted as (avg. latency *  
  in-flight requests) according to Little's Law. The proportion of incoming tokens  
  that are admitted is dictated by the signal at the `load_multiplier` port.  
  
  :::info See Also  
  
  Load Scheduler [[Reference](https://github.com/fluxninja/aperture/pull/2055)](/reference/policies/spec.md#load-scheduler)  
  
  :::  
- ### Adaptive Load Scheduler {#adaptive-load-scheduler}
  
  Building upon the foundation of the *Load Scheduler*, the *Adaptive Load
  Scheduler* fine-tunes the request management process. It incorporates a gradient  
  controller and integrator for computing the load multiplier, which adjusts based  
  on a signal, setpoint, and overload signals. The key functionality of the  
  *Adaptive Load Scheduler* lies in its ability to adjust the accepted token rate  
  based on the deviation of the input signal from the setpoint. This adaptability  
  allows for greater control and precision in managing load.  
  
  :::info See Also  
  
  Adaptive Load Scheduler  
  [[Reference](https://github.com/fluxninja/aperture/pull/2055)](/reference/policies/spec.md#adaptive-load-scheduler)  
  
  :::  
- ### Quota Scheduler {#quota-scheduler}
  
  The Quota Scheduler uses a global-level token bucket that functions as a ledger.  
  This ledger maintains an account of the total available tokens that can be  
  allocated across all agents. In a scenario where the total load or the total  
  number of tokens is known upfront, the Quota Scheduler can be particularly  
  effective. These Tokens can be thought of as a fixed quota that is distributed  
  among the agents. Each agent has access to this global ledger and deducts tokens  
  from it when processing requests. If the ledger runs out of tokens, new requests  
  are queued or rejected until more tokens become available. The Quota Scheduler  
  is useful in scenarios where there are strict global rate limits, for example,  
  when scheduling access to external APIs that have a fixed rate limit.

Reply:
Workloads are a property of Scheduler. They can be defined in quota scheduler too and allow prioritizing requests when hitting quota limits